### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by andrewbaxter** [dmnd/git-diff-blame#14]
The readme doesn't mention it and I think it would be useful for people less familiar with installing git extensions.  I think you just need to put it somewhere on PATH, but not 100% sure.

**Comment by hlovdal** [comment]
Yes, either copying the script to a directory already included in `PATH` environment variable, or add the directory it is present in to `PATH`. Potentially also `chmod +x git-diff-blame`.
